new quickstart guides (#3060)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/createproject.md

> You will need recent versions of [Node](https://nodejs.org/en/) and  [npm](https://www.npmjs.com/). Installation of [Angular CLI](https://angular.io/cli) is recommended.

Reviewer: ```suggestion
You need recent versions of [Node](https://nodejs.org/en/) and  [npm](https://www.npmjs.com/). We recommend that you install the [Angular CLI](https://angular.io/cli).
```

> Create an Angular application named "okta-angular-quickstart" with routing using the following command. This creates an Angular application using version 13 regardless of the version of the Angular CLI installed on your system.

Reviewer: ```suggestion
Create an Angular application named "okta-angular-quickstart" with routing using the following command. This creates an Angular application using version 13 regardless of the version of the Angular CLI installed on your system.
```
```suggestion
Create an Angular application with routing named "okta-angular-quickstart" using the following command. This creates an Angular application using version 13 regardless of the version of the Angular CLI installed on your system.
```
